src/Serilog.Sinks.Map/MapLoggerConfigurationExtensions.cs

```diff
@@ -44,13 +53,21 @@ public static class MapLoggerConfigurationExtensions
             this LoggerSinkConfiguration loggerSinkConfiguration,
             string keyPropertyName,
             Action<string, LoggerSinkConfiguration> configure,
-            string defaultKey = null,
+            string defaultKey,
```

Comment:
Perhaps, if this is mandatory now, we move it before `configure` so that we don't have two overloads that differ only by delegate signature at the same parameter position? (This can be annoying when there are syntax errors/partial implementations, because the compiler can't generate useful type errors.)

Reply:
Didn't thought about this, good point.